lab-sarah/ng-template/app/component/gallery/thumbnail-container/thumbnail-container.html

```diff
@@ -0,0 +1,12 @@
+<div class="thumbnail-container">
+  <h2> {{thumbnailContainerCtrl.gallery.name}} </h2>
+  <upload-pic gallery="thumbnailContainerCtrl.gallery"></upload-pic>
+  <searchbar
+  search-assets="search by name"
+  search-term="homeCtrl.pictureNameSearchTerm"></searchbar>
+  <searchbar
```

Comment:
Good work adding your searchbar component to your html and passing the correct bindings!